docs/source-datastore/scan.md

```diff
@@ -276,47 +300,53 @@ The activity heatmap shown in the snippet below represents activity levels over
-| Summary | The summary section provides an overview of the scan operation up to the point it was aborted. It includes: <br> <ul><li> **Tables Requested**: The total number of tables that were scheduled for scanning. Click on the adjacent magnifying glass icon to view the tables requested. </li><li> **Tables Scanned**: The number of tables that have been scanned so far. Click on the adjacent magnifying glass icon to view the tables scanned. </li><li> **Partitions Scanned**: The number of partitions scanned before the operation was aborted.</li><li> **Records Scanned**: The total number of records processed before the scan was stopped. </li><li> **Anomalies Identified**: The number of anomalies detected during the partial scan. </li></ul>|
-
 ![running](../assets/datastores/scan/running-light.png#only-light)
 ![running](../assets/datastores/scan/running-dark.png#only-dark)
 
+| No. | Parameter                  | Interpretation                                                                                               |
+| --- | -------------------------- | ----------------------------------------------------------------------------------------------------------- |
+| 1   | Operation ID and Type   | Unique identifier and type of operation performed (catalog, profile, or scan).                               |
+| 2   | Timestamp              | Timestamp when the operation was started.                                                                    |
+| 3   | Progress Bar            | The progress of the operation.                                                                               |
+| 4   | Triggered By            | The author who triggered the operation.                                                                      |
+| 5   | Schedule                | Indicates whether the operation was scheduled or not.                                                        |
+| 6   | Incremental Field       | Indicates whether Incremental was enabled or disabled in the operation.                                      |
+| 7   | Remediation             | Indicates whether Remediation was enabled or disabled in the operation.                                      |
+| 8   | Anomalies Identified    | Provides a count of the number of anomalies detected before the operation was aborted.|
+| 9   | Read Record Limit        | Defines the maximum number of records to be scanned per table after initial filtering.|
+| 10  | Check Categories        | Indicates which categories should be included in the scan (e.g., Metadata, Data Integrity).|
+| 11  | Archive Duplicate Anomalies | Indicates whether Archive Duplicate Anomalies was enabled or disabled in the operation.
+| 12  | Source Record Limit     | Indicates the limit on records stored in the enrichment datastore for each detected anomaly.|
+| 13  | Results                 | View the details of the scan operation that was aborted, including scanned tables and identified anomalies.|
```

Comment:
On line 320 Results - Was explaining the `Aborted` operation. But in this part we are talking about `Running` operation

Reply:
On line 320, the Results section has been updated to explain the Running operation instead of the Aborted operation.